granary/reddit.py

```diff
@@ -0,0 +1,215 @@
+# coding=utf-8
+"""reddit.com source class.
+"""
+
+from . import source
+import datetime
+import logging
+from oauth_dropins import reddit
+from oauth_dropins.webutil import util
+import re
+import urllib.parse, urllib.request
+
+import praw
+
+class Reddit(source.Source):
+
+    DOMAIN = 'reddit.com'
+    BASE_URL = 'https://reddit.com'
+    NAME = 'reddit'
+
+    def __init__(self, refresh_token):
+        self.refresh_token = refresh_token
+
+    @staticmethod
+    def timestamp_to_iso8601(time_stmp):
+        """Converts a timestamp from UNIX format to ISO 8601.
+        """
+        if not time_stmp:
+            return None
+
+        return datetime.datetime.fromtimestamp(time_stmp).isoformat()
+
+    def redditor_to_actor(self, user):
```

Comment:
these methods generally have common names from `Source`. this one is `user_to_actor`:

https://github.com/snarfed/granary/blob/6e97787c72a139709e011587a65722ec64212fe4/granary/source.py#L510-L523